types/index.d.ts

```diff
@@ -1,5 +1,8 @@
+// TypeScript Version: 3.0
```

Comment:
@jquense it was the absence of these version comments – dtslint was trying to validate against ts 2 (!)

Reply:
are they going to move dtslint off of tslint? hopefully in the future we'd just use eslint

Reply:
this isn't the linting stuff per se – this is the actual running of the type checking

that's why dtslint installs like every typescript version. it wants to check for backward compatibility of the type defs

Reply:
you are saying that that comment `// TypeScript Version: 3.0` isn't a dtslint thing but it's actually used by tsc?

Reply:
i don’t know exactly how it works, but the failures were tsc failures, not tslint ones. maybe it’s a hint to dtslint for which versions of tsc to run

Reply:
yeah I think so